spring-boot-starters/langchain4j-open-ai-spring-boot-starter/src/test/java/dev/langchain4j/openai/spring/AutoConfigIT.java

```diff
@@ -0,0 +1,176 @@
+package dev.langchain4j.openai.spring;
+
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.model.language.LanguageModel;
+import dev.langchain4j.model.moderation.ModerationModel;
+import dev.langchain4j.model.openai.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.springframework.boot.autoconfigure.AutoConfigurations;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import static dev.langchain4j.model.openai.OpenAiModelName.GPT_3_5_TURBO;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@EnabledIfEnvironmentVariable(named = "OPENAI_API_KEY", matches = "sk.*")
```

Comment:
What is this "sk" prefix? If this comes from Semantic Kernel, I would wait a bit before copying them as their Spring Boot starter isn't finished yet

Reply:
No, this is an OpenAI's API key prefix, I guess it stands for secret key